homework_6/задания 3 уровня/5-имитация_загрузки_картинок/load.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="products">
+        <div class="product">
+            <div class="imgWrap">
+                <img src="images/loading.gif" alt="">
+            </div>
+            <div>Товар 1</div>
+            <div>33.88 р.</div>
+        </div>
+        <div class="product">
+            <div class="imgWrap">
+                <img src="images/loading.gif" alt="">
+            </div>
+            <div>Товар 2</div>
+            <div>240.41 р.</div>
+        </div>
+        <div class="product">
+            <div class="imgWrap">
+                <img src="images/loading.gif" alt="">
+            </div>
+            <div>Товар 3</div>
+            <div>971.36 р.</div>
+        </div>
+    </div>
+
+    <!-- Задание. Нужно через 3 секунды заменить все картинки loading.gif на картинки
+    из папки images.
+    1. Получите все картинки например с помощью querySelectorAll
+    2. Переберите эту коллекцию с помощью цикла for, в цикле каждой картинке
+    установите атрибут src (например с помощью setAttribute) равный 
+    "images/здесь_нужно_поставить_правильное_имя_файла".
+    3. Наконец весь код оберните в setTimeout, и выставите задержку 3 сек.
+    -->
+
+    <script>
+        let imgList = document.querySelectorAll("img");
+        setTimeout(() => {
+            for (let img of imgList) {
+                for (let i = 0; i < imgList.length; i++) {
+                    img.setAttribute("src", `images/${[].indexOf.call(imgList, img) + 1}.jpg`);
+                }
+            }
```

Comment:
Зачем тебе цикл внутри цикла? Да еще и индекс элемента ищешь потом функцией indexOf, которая тоде циклом идет по всему массиву.
Я конечно могу строить теории зачем ты так сделал, но все гораздо проще, если задача кажется запутанной, попробуй отойти на шаг назад и еще раз подумать
```suggestion
            for (let i = 0; i < imgList.length; i++) {
                imgList[i].setAttribute("src", `images/${i + 1}.jpg`);
            }
```